src/main/resources/assets/thaumcraft/lang/en_us.lang

```diff
@@ -6,6 +6,11 @@ potion.unhunger=Unhunger
 # BASICS
 ##########################################
 #
+got.deepdown=§5You have visited deep and dark depths and have learned much.
```

Comment:
I think it's possible to compose TextComponents together, so you could send a TextComponent for setting the color to purple plus a translation component together. It's not worth making you make another round of changes for, so I'll keep this for now and check that myself later